content/fr/docs/concepts/architecture/cloud-controller.md

```diff
@@ -0,0 +1,264 @@
+* Service controller
+* PersistentVolumeLabels controller
+
+#### Node controller
+
+Le Node controller est responsable de l'initialisation d'un node en obtenant des informations sur les nodes s'exécutant dans le cluster dans le cadre d'un provider cloud. Le node controller exécute les fonctions suivantes:
+
+1. Initialisez un node avec des labels de zone/région spécifiques au cloud.
+2. Initialisez un node avec des paramètres d'instance spécifiques au cloud, par exemple, le type et la taille.
+3. Obtenez les adresses réseau et le nom d'hôte du node.
+4. Si un node ne répond plus, vérifiez que l'instance n'a pas ete suprimée sur votre cloud provider.
+Si le node a été supprimé du nuage, supprimez l'objet Kubernetes Node.
+
+#### Route controller
+
+Le Route controller est responsable de la configuration appropriée des routes réseau dans le cloud afin que les conteneurs situés sur différentes node du cluster Kubernetes puissent communiquer entre eux. Le Route controller ne s'applique qu'aux clusters Google Compute Engine.
+
+#### Service Controller
+
+Le Service controller est chargé d'écouter les événements de création, de mise à jour et de suppression de service. Sur la base de l'état actuel des services dans Kubernetes, il configure les équilibreurs de charge cloud (tels qu'un ELB ou Google LB) pour refléter l'état des services dans Kubernetes. De plus, cela garantit que les services de base des services pour les équilibreurs de charge dans le cloud sont à jour.
+
+#### PersistentVolumeLabels controller
+
+Le PersistentVolumeLabels controller applique les labels sur les volumes AWS EBS/GCE PD lors de leur création. Cela évite aux utilisateurs de définir manuellement les labels sur ces volumes.
+
+Ces labels sont essentielles pour le scheduling des pods car ces volumes ne doivent fonctionner que dans la région/zone dans laquelle ils se trouvent. Tout pod utilisant ces volumes doit être schedulé dans la même région/zone.
+
+Le PersistentVolumeLabels controller a été créé spécifiquement pour le CCM; c'est-à-dire qu'il n'existait pas avant la création du CCM. Cela a été fait pour déplacer la logique de labelisation des PV dans le serveur API Kubernetes (il s'agissait d'un admission controller) vers le CCM. Il ne fonctionne pas sur le KCM.
+
+### 2. Kubelet
+
+Le Node controller contient les fonctionnalités de Kubelet dépendantes du cloud. Avant l’introduction du CCM, la sous-unité était responsable de l’initialisation de nodes avec des paramètres spécifiques au cloud, tels que les adresses IP, les étiquettes de région/zone et les informations de type d’instance. L’introduction du CCM a déplacé cette opération d’initialisation de Kubelet au CCM.
+
+Dans ce nouveau modèle, Kubelet initialise un node sans paramètres spécifiques au cloud. Cependant, il ajoute un tag taint au node nouvellement créé, ce qui le rend non modifiable jusqu'à ce que le CCM initialise le node avec les informations spécifiques au cloud. Il supprime ensuite ce tag taint.
```

Comment:
suggest tag -> label, or :"cette teinte"

Reply:
same as "fournisseurs de clouds"

Reply:
label yes

Reply:
label is slightly different no? it's more accurate to use "teinte" there i believe... but i could be wrong